db/1.14/modules/ROOT/pages/database-connector-troubleshooting.adoc

```diff
@@ -0,0 +1,299 @@
+= Troubleshooting Database Connector - Mule 4
+
+To troubleshoot Anypoint Connector for Database (Database Connector), become familiar with the information about enabling verbose logging, troubleshooting pooling profile connections, troubleshooting Snowflake JDBC driver, and interpreting commonly thrown messages.
+
+Database Connector follows the standards and guidelines detailed by the JDBC library (working with Java SE JDK 1.8).
+
+== Enable Verbose Logging
+
+To get a better understanding of why an application's interaction with the database is failing, temporarily enable verbose logging for Database Connector. +
+Choose one of two options to enable verbose logging: manually change the `log4j2.xml` file or via Anypoint Runtime Manager if you are using CloudHub APIs. +
```

Comment:
I would use bullets for the two options because it is easier to scan them. Can we tell them how to change the log4j2.xml file? If I am understanding this correction, I suggest:

Manually change the `log4j2.xml` file by xxx.
If you are using CloudHub APIs, use Anypoint Runtime Manager to change xxx by yyy.